src/chapters/html/tags.rst

```diff
@@ -0,0 +1,83 @@
+HTML Tags
+=========
+
+Time to dive into learning about all the different tags for creating content!
+This page contains a helpful table of tags to know for beginning programmers to bookmark.
+This is by no means an exhaustive list of all HTML tags, but it is a good place to start.
+
+Tags to Know
+------------
+
+.. list-table::
+   :widths: auto
+   :header-rows: 1
+
+   * - Tag Name
+     - Code
+     - Definition
+   * - Bold
+     - ``<b>``
+     - When surrounding text, makes that text bold.
+   * - Emphasis
+     - ``<em>``
+     - When surrounding text, makes that text italic.
+   * - Hyperlink
+     - ``<a>``
+     - Creates hyperlinks.
+   * - Image
+     - ``<img>``
+     - Denotes images.
+   * - Break
+     - ``<br>``
+     - A single line break.
+   * - Paragraph
+     - ``<p>``
+     - Creates a paragraph in text.
+   * - Section
+     - ``<span>``
+     - Makes a section in text.
+   * - Division
+     - ``<div>``
+     - Defines an area of the page.
+   * - Form
+     - ``<form>``
+     - Creates a form for user input.
+   * - Unordered List
+     - ``<ul>``
+     - Creates an unordered list.
+   * - Ordered List
+     - ``<ol>``
+     - Creates an ordered list.
+   * - List element
+     - ``<li>``
+     - Denotes an element of the list. This tag is used for both ordered and unordered lists.
+   * - Table
+     - ``<table>``
+     - Creates a table on the page.
+
+Attributes
+----------
+
+.. index:: ! attribute
+
+Programmers can add extra information beyond element type to HTML tags.
+Programmers add **attributes** to HTML tags for further specification about the element's appearance.
+Examples of attributes include the alignment of the element or alternate text to an image.
+
+Programmers add attributes before the closing bracket in the opening tag, like so:
+
+.. sourcecode:: html
+   
+   <element attribute = "value">content</element>
+
```

Comment:
What do you think about having a few examples of common elements with the html and an image of what the browser would render?